sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -38,82 +37,147 @@
 
 
 class SdkHarness(object):
+  REQUEST_METHOD_PREFIX = '_request_'
+
+  def __init__(self, control_address, pipeline_options=None):
```

Comment:
<!--thread_id:cc_155909205_t; commit:cde0dc6b3bc26cce4c845c77bac82df558635058; resolved:0-->
<!--section:context-quote-->
> **angoenka** wrote:
> As we have the default value of worker_threads, the SdkHarness can work without pipeline_options.
> But as mentioned in the below comment, I will extract the thread count in the sdk_worker_main and pass the thread_count to the sdk_harness. I will also rename it to worker_count to avoid usage of different terms (thread_count and worker_count) in SdkHarness.

<!--section:body-->
Since this is simplified to just `worker_count`, I am in favor not making it an optional argument and passing an explicit value in other callers. (I assume other places are generally tests.)

Reply:
Yup, other places are mostly tests. sdk_worker_main is the only production code which touches creates sdk_worker_harness.
I will default the value from the caller side in that case.